src/v2/Components/NavBar/NavBar.tsx

```diff
@@ -149,7 +150,7 @@ export const NavBar: React.FC = track(
                   flex={1}
                   size="small"
                 >
-                  Sign Up
+                  <FormattedMessage id="navbar.signup" />
```

Comment:
i still feel very strongly that this should not look like `<FormattedMessage id="navbar.signup" />`, and instead should look like `t("navbar.signup")` or something very very minimal.

Reply:
Yeah, this is kinda confusing. There's a hook that can be used which seems nicer.

Reply:
Agree with @pvinis! Being locked into an react component for this can be limiting here since we're often using logic in plain js -- ` foo ? "hello" : "world". That kind of thing. 

But that said, i could be convinced that a react component like this is indeed better. Worth a bit of discussion.

Reply:
An implementation like this doesn't seem to be this straightforward with `react-intl`, but it's the natural way to go in `react-i18next`.

There is this other PR here with an attempt of using `react-i18next`: https://github.com/artsy/force/pull/9797